common/attestation.edl

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** attestation.edl:
+**
+**     This file declares internal ECALLs used by liboehost/liboecore for
+**     gathering/verifying attestion information from the enclave.
+**
+**==============================================================================
+*/
+
+enclave
+{
+    include "openenclave/bits/types.h"
```

Comment:
I think this header is not needed, can you double check?

Reply:
oh ya you're right. I got a little copy/paste happy here